ContestModel/src/org/icpc/tools/contest/model/feed/DiskContestSource.java

```diff
@@ -1101,9 +1102,15 @@ private FileReferenceList getFilesWithPattern(FilePattern pattern) {
 					String diff = file.getName();
 					diff = diff.substring(pattern.name.length(), diff.length() - ext.length() - 1);
 					FileReference ref = getMetadata(pattern.url + diff, file);
-					if (ref != null)
+					if (ref != null) {
+						// update the href if it is already in use (e.g. for logo.svg and logo.png)
+						int count = 2;
```

Comment:
We should increase count, but isn't #1115 better?

Reply:
Oh, I missed the change to line 1104 and thought the draft status meant it wasn't quite done. Yeah, that works too, but I think we should pick an option that can't have conflicts and hence doesn't need the warning output, plus remove the TODO.

I had leaned toward not including the file extension in the href originally so was using the counter instead. I think it would probably be ok to just use the real filename too instead of either of these changes, just less pretty hrefs.